scripts/assumeRole.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
Do we really need this script in 2 repos now? I think we should all be using the GDS CLI. However, if people still prefer this method then I guess it's fine!

Reply:
Should this be raised during standup?